### PR TITLE
ENG-973 Integration the github starring in the CLI

### PR DESCRIPTION
**Pull request by zreigz**
## Summary
After successful init, the user will be asked if he would like `Star` Plural repository if he did not.

## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [ ] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: pkg/scm/gitlab.go [hunk 1]
```diff
@@ -36,6 +37,7 @@ func (gl *Gitlab) Init() error {
 
 	git, err := gitlab.NewOAuthClient(accessToken.Token)
 	gl.Client = git
+	gl.Token = accessToken.Token
```
Comment (michaeljguarino) [inline pkg/scm/gitlab.go:40]: this token should expire pretty quickly, like in a matter of hours

Comment (michaeljguarino) [reply pkg/scm/gitlab.go:40]: a better solve is probably to just do the github star process in-line with plural init or something like that

Comment (zreigz) [reply pkg/scm/gitlab.go:40]: @michaeljguarino  It was my first thought too, but I found this: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/token-expiration-and-revocation#token-expired-due-to-lack-of-use

so the token is valid for one year

Comment (zreigz) [reply pkg/scm/gitlab.go]: done

**Comment by zreigz** [comment]
@michaeljguarino PTAL, I have simplified the code and moved it to the init command

File: pkg/scm/github.go [hunk 2]
```diff
@@ -113,3 +114,29 @@ func (gh *Github) Setup() (con Context, err error) {
...
+func (gh *Github) StarPluralGitHubRep() error {
+	ctx := context.Background()
+	starred, _, err := gh.Client.Activity.IsStarred(ctx, "pluralsh", "plural")
+	if err != nil {
+		return err
+	}
+	if !starred {
+		proceed := false
+		prompt := &survey.Confirm{
+			Message: "Would you like to Star us on the GitHub",
```
Comment (michaeljguarino) [inline pkg/scm/github.go:127]: let's change this message to `Would you like to show your support for the project with a github star?`

**Review by michaeljguarino — APPROVED** [review]
small change but otherwise lgtm

**Pull request merged**